### PR TITLE
new(account/user mgmt tutorial): new doc

### PR DESCRIPTION
**Pull request by zuluecho9**
Work in progress. DOC-7841

### CODE REVIEW

**Review by rhetoric101 — APPROVED** [review]
@zuluecho9 thanks for adding these docs. I know you'll be doing some fast-follow improvements.

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial.mdx [hunk 1]
```diff
@@ -0,0 +1,207 @@
+---
+title: 'A tutorial on setting up accounts, access grants, and users'
```
Comment (bradleycamacho) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial.mdx:2]: This feels like a weird title. Maybe `Setting up accounts, access grants, and users tutorial`.

Not super happy with that either, but just an idea.

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial.mdx [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+<img
+  title="New Relic organization and access UI - default access grants"
+  alt="New Relic organization and access UI - default access grants"
+  src={orgAccessUI}
+/>
+
+You can see how the **User** group has the **All product admin** role and access to that initially created account. And you can see how the **Admin** group has four access grants, reflecting its greater capabilities. The "Default" next to those group names refers to them being in the original, default authentication domain (we'll talk more about that later). 
+
+Pro and Enterprise edition organizations are able to add custom groups, or bring in groups from their identity provider.
```
Comment (bradleycamacho) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial.mdx:59]: Is there a doc we can link out to for this? Seems like a complex topic and a bit weird to leave at just that.

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial.mdx [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+Pro and Enterprise edition organizations are able to add custom groups, or bring in groups from their identity provider. 
+
+## Step 2: Add accounts [#add-accounts]
+
+Before adding your users in New Relic, you might want to set up some accounts and [get some data reporting](/docs/using-new-relic/welcome-new-relic/get-started/get-started-full-stack-observability) to them. All of that isn't required at this point (you can always add more accounts later and grant users access to them later) but for the purposes of this tutorial, we'll walk you through adding accounts. 
+
+As covered in Step #1, an account is meant to represent a workspace of some sort. It's a useful way to separate your use of New Relic by business-relevant boundaries. An account can have a large amount and variety of data reporting to it. There's nothing preventing even quite large companies from having a single account or just a handful of accounts. It's all really a matter of what your organization's goals are and how useful you find it to create the account boundaries.
```
Comment (bradleycamacho) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial.mdx:65]: Should `Step #1` be capitalized?

File: src/content/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign.mdx [hunk 4]
```diff
@@ -45,8 +45,8 @@ For an explanation of how your identity provider groups map over to New Relic gr
 
 To use automated user management to import users from your identity provider:
 
-1. It's important to first review the [requirements](#requirements).
-2. In the [authentication domain UI](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more/#ui), create a new authentication domain.
+1. Recommended: first review the [requirements](#requirements).
```
Comment (bradleycamacho) [inline src/content/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign.mdx:48]: I tend to use `step 0:` when something is optional or recommended

**Pull request merged**